scripts/build/tests.js

```diff
@@ -167,23 +156,26 @@ exports.cleanTestDirs = cleanTestDirs;
  * @param {boolean} [keepFailed]
  * @param {number | undefined} [shards]
  * @param {number | undefined} [shardId]
+ * @param {boolean} [color]
  */
-function writeTestConfigFile(tests, runners, light, taskConfigsFolder, workerCount, stackTraceLimit, timeout, keepFailed, shards, shardId) {
+function writeTestConfigFile(reporter, reporterOption, tests, runners, light, taskConfigsFolder, workerCount, stackTraceLimit, timeout, keepFailed, shards, shardId, color) {
     const testConfigContents = JSON.stringify({
-        test: tests ? [tests] : undefined,
-        runners: runners ? runners.split(",") : undefined,
+        reporter,
+        reporterOption,
+        grep: tests || undefined, // tests is a string interpreted as a regexp
+        runners: runners ? runners.split(/ *, */) : undefined,
         light,
         workerCount,
         stackTraceLimit,
         taskConfigsFolder,
-        noColor: !cmdLineOptions.colors,
+        color,

```

Comment:
Changing this name will impact the parallel test runner. You will also need to change the name (and invert the value) in the following two locations:

https://github.com/microsoft/TypeScript/blob/5d3284032b62748da5b8e7dd4fa8fca3ae8141a4/src/testRunner/runner.ts#L81

https://github.com/microsoft/TypeScript/blob/5d3284032b62748da5b8e7dd4fa8fca3ae8141a4/src/testRunner/runner.ts#L111-L113